help2man.lua

```diff
@@ -0,0 +1,154 @@
+
```

Comment:
Wow.. I guess there is no reason not to include a lua script here.. most folks won't ever need to run it.

Can you put this in the scripts/ directory?   And maybe add a `#!` to the top?

Reply:
aw yeah... i should really use my linux laptop more ...
... done